CSharpOOP4/CSharpOOP4/Ball.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP4
+{
+    public class Ball : Shape
+    {
+        private double _radius;
+        double ballVolume;
```

Comment:
access modifier. And add space between fields, properties and constructor. Fix for all classes.